### DISCUSSION TITLE
When clicking on a PR. It always says something went wrong in the latest update.

### DISCUSSION THREAD

**Discussion by Fujiwara16** [community/community#65581] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.128.0 (108490092)
Device: iPhone 12 (iOS 16.6)
</details>

**Comment by alcere** [comment]
Hi @Fujiwara16, we are sorry for the troubles! I will let the team know and create an internal issue to follow up on this, thanks!
